client/pages/students/[student_id]/index.tsx

```diff
@@ -16,10 +17,13 @@ const ViewStudentPage = () => {
 
   return (
     <div>
-      <h1>Student {student?.student_id}</h1>
+      <h1>Student: {student?.student_id}</h1>
       <p>
         {student?.first_name} {student?.last_name}
       </p>
+      <button>
+        <Link href={"/students"}>GO BACK</Link>
```

Comment:
In general back/forward is associated with browser navigation. Maybe change this to "Go to full list" or similar?

Reply:
> In general back/forward is associated with browser navigation. Maybe change this to "Go to full list" or similar?

Yes, I can change it to "Return to Student List". Also, I'm not quite sure if the test failure is my portion or not.

Reply:
I believe it was introduced on the push to main.

Open PR to fix: https://github.com/sfbrigade/project-compass/pull/38